src/plugins/BunqLayoutPlugin/components/BunqLayout/BunqLayout.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+/**
+ * @author Nicola Parrello <nparrello@bunq.com>
+ * @since  20181005 Initial creation.
+ */
+class BunqLayout extends React.Component {
+    /**
+     * @returns {Node}
+     */
+    render (): Node {
+        const { getComponent } = this.props;
+        const BaseLayout = getComponent("BaseLayout", true);
+
+        return (
+            <div className="wrapper">
+                {
+                    this.renderSidebar()
+                }
+                <BaseLayout />
+            </div>
+        );
+    }
+
+    /**
+     * @returns {Node}
+     */
+    renderSidebar (): Node {
```

Comment:
I prefer to always have the original `render()` at the bottom of your file because I automatically look at the bottom when I'm looking for it. but your choice I guess, as long as you keep it consistent in the future